pilot/pkg/security/authn/v1beta1/policy_applier.go

```diff
@@ -272,11 +321,17 @@ func convertToEnvoyJwtConfig(jwtRules []*v1beta1.JWTRule, push *model.PushContex
 						CacheDuration: &durationpb.Duration{Seconds: 5 * 60},
 					},
 				}
-			} else {
+			} else if remoteJwksMode == Hybrid {
 				provider.JwksSourceSpecifier = push.JwtKeyResolver.BuildLocalJwks(jwtRule.JwksUri, jwtRule.Issuer, "")
```

Comment:
Do we need to provide jwksInfo to Istiod?

Reply:
not needed from what I understand.